Dockerfile

```
@@ -0,0 +1,63 @@
+COPY tsconfig.json ./
+
+# Build TypeScript to JavaScript
+RUN bun run build
+
+# ============================================
+# Stage 2: Production - Minimal runtime image
+# ============================================
+FROM oven/bun:1.3-alpine AS production
+
+# Install dumb-init for proper signal handling
+RUN apk add --no-cache dumb-init
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)